Description: meson 0.58 is more strict with mixed lists and will not
automatically flatten lists.
Results in:

> tests/meson.build:125:4: ERROR: List item must be one of <class 'str'>, not <class 'list'>


diff --git a/tests/meson.build b/tests/meson.build
index 4696b04..4e25079 100644
--- a/tests/meson.build
+++ b/tests/meson.build
@@ -125,8 +125,8 @@ if get_option('compose')
     test ('as-test_compose',
         as_test_compose_exe,
         args: as_test_args,
-        env: [as_test_env,
-              'LSAN_OPTIONS=suppressions=' +
+        env: as_test_env +
+              ['LSAN_OPTIONS=suppressions=' +
                   join_paths(meson.current_source_dir(), 'lsan-suppr.txt')]
     )
 endif
